### PR TITLE
Type field

### PR DESCRIPTION
**Pull request by Akshay-Jak**
### Functionality:
Added a New Field called Type

### Solution:
Added a new field Type to know the type of the task that is being assigned

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start

### CODE REVIEW

File: src/app/body/create-new-session/create-new-session.component.html [hunk 1]
```diff
@@ -108,6 +108,17 @@ <h5 class="active nav-link" id="head">Create New Task</h5>
...
                 </div>
+                <div class="form-row">
+                    <div class="form-group col-md-3">
+                        <label for="input" class="form-label">Type</label>
+                        <input type="text" class="form-control" [(ngModel)]="type" [ngModelOptions]="{standalone: true}" list="typeSuggestion">
+                        <datalist id="typeSuggestion">
+                            <option>Bug</option>
+                            <option>Feature</option>
+                            <option>UI</option>
+                        </datalist>
```
Comment (viv9k) [inline src/app/body/create-new-session/create-new-session.component.html:119]: Make this list dynamic like others

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

**Comment by k0malSharma** [comment]
Resolve merge conflict in _**functions/model/tasks/createTask.js**_ as well

**Review by viv9k — CHANGES_REQUESTED** [review]
At some places I see label and at some places it's type.

Use one everywhere, best convert everything to type.

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**